### DISCUSSION TITLE
Tag user or role on discord

### DISCUSSION THREAD

**Discussion by murilomorato** [sarisia/actions-status-discord#387] [Ideas, Feature requests]
I created a server to receive alerts whenever a deploy fails, but I haven't been receiving notifications alerts.
I tried to insert inside the 'description' prop my Role tag and my user id. They arrive tagged in the discord message, but it does not generate the notification, as I believe the message arrives as a kind of 'quote', not as a normal message.

Have you tried any alternatives yet? 

Thanks😉

**Comment by sarisia** [comment]
Nice catch!

I can reproduce this and also found the related Reddit post: https://www.reddit.com/r/discordapp/comments/kffg90/mention_inside_embed_with_ping/

Confirmed this generates ping to the user:

```json
{
  "content": "<@316911818725392384>",
  "embeds": [{
    "description": "<@316911818725392384>"
  }]
}
```

while this doesn't (no content field):

```json
{
  "embeds": [{
    "description": "<@316911818725392384>"
  }]
}
```

Sadly we cannot append the `content` field for now, but we obviously need this so I'll implement this as a new action input soon!

**Comment by sarisia** [comment]
This is now released at [v1.11.0](https://github.com/sarisia/actions-status-discord/releases/tag/v1.11.0)!

Thank you for feature suggestion!
